requirements/docs.txt

```diff
@@ -1,3 +1,4 @@
 sphinx==2.4.4
 sphinx-rtd-theme==0.4.3
-m2r==0.2.1
\ No newline at end of file
+m2r==0.2.1
+jsonschema==3.2.0
```

Comment:
I think it should be removed.

Reply:
Talked offline. jsonschema is needed to render docstrings from `optimizely`, `config_manager` , and `optimizely_config`.